spec/components/sdg/related_list_selector_component_spec.rb

```diff
@@ -5,6 +5,11 @@
   let(:form) { ConsulFormBuilder.new(:debate, debate, ActionView::Base.new, {}) }
   let(:component) { SDG::RelatedListSelectorComponent.new(form) }
 
+  before do
+    Setting["feature.sdg"] = true
+    Setting["sdg.process.debates"] = true
```

Comment:
It would be great to add some tests to check when this component is (not) rendered.